.gitignore

```diff
@@ -1 +1,3 @@
 /vendor/
+/notes
+.*.sw?
```

Comment:
I think this is specific to your local setup and so should be ignored locally (either per project or in the global git config), not in the project: https://help.github.com/articles/ignoring-files/